cloud/kubernetes/helm/templates/pvc.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if .Values.persistence.scSupportsReadWriteMany }}
 {{- range $i, $vName := tuple "data-store" "logs" "rms-shared" "rms-security" "rms-webstudio" }}
-{{- if or (and (eq $vName "data-store") (eq $.Values.bsType "sharednothing")) (and (eq $vName "logs") $.Values.persistence.logs) (and (eq $vName "rms-shared") $.Values.rmsDeployment) (and (eq $vName "rms-security") $.Values.persistence.rmsSecurity $.Values.rmsDeployment) (and (eq $vName "rms-webstudio") $.Values.persistence.rmsWebstudio $.Values.rmsDeployment) }}
```

Comment:
$.Values.persistence.rmsWebstudio $.Values.rmsDeployment) is repeated twice and rms-security condition is also there